test/junit/scala/PredefTest.scala

```diff
@@ -20,4 +20,20 @@ class PredefTest {
     assertEquals("bazzz", c)
     assertEquals(5, res)
   }
+
+  @Test
+  def testAnyTap: Unit = {
+    var x: Int = 0
+    val result = List(1, 2, 3)
+      .tap(xs => x = xs(0))
```

Comment:
Not a big deal, but isn't it a little non-idiomatic to index into a list instead of taking the head or something?